Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -187,4 +187,15 @@ final class MapViewTests: XCTestCase {
         mapView.didReceiveMemoryWarning()
     }
 
+    func testWillEnterForeground() {
+        mapView.willEnterForeground()
+
+        XCTAssertFalse(displayLink.isPaused)
+    }
+
+    func testDidEnterBackground() {
+        mapView.didEnterBackground()
+
+        XCTAssertTrue(displayLink.isPaused)
+    }
```

Comment:
You can stub the isPaused setter to enable verifying that the setter was actually invoked vs. the property just accidentally having the expected value already.

Reply:
Stubbed